src/common/utils/model.ts

```diff
@@ -100,7 +100,14 @@ export function fetchAllStateProjections(snapshotUrl: string | null = null) {
 
 /** Returns an array of `Projections` instances for all counties. */
 const cachedCountiesProjections: { [key: string]: Promise<Projections[]> } = {};
-export function fetchAllCountyProjections(snapshotUrl: string | null = null) {
+export function fetchAllCountyProjections(
+  snapshotUrl: string | null = null,
+  queryByState: boolean = true,
```

Comment:
I don't think we really need this flag.  We can just merge these methods and always query by state.